tools/functional-tester/etcd-runner/command/lock_racer_command.go

```diff
@@ -69,12 +72,16 @@ func runRacerFunc(cmd *cobra.Command, args []string) {
 		m := concurrency.NewMutex(s, racers)
 		rcs[i].acquire = func() error { return m.Lock(ctx) }
 		rcs[i].validate = func() error {
+			mu.Lock()
```

Comment:
how is this functionally different from what's in the code now?

The current code:
```go
mu.Lock()
f()
mu.Unlock()
```

does the same thing as this new code:
```go
func f() {
    mu.Lock()
    defer mu.Lock()
    ...
}
```
(panics not withstanding)